src/components/PlayerCardContainer/PlayerCard.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from "react";
+
+import CustomInput from "./CustomInput";
+import CustomButton from "./CustomButton";
+import './PlayerCard.css'
+
+
+function PlayerCard(props) {
+
+    function nameHandler(event) {
+        props.onNameChange(event, props.player.id)
+    }
+
+    function balanceHandler(event) {
+        props.onBalanceChange(event, props.player.id)
+    }
+
+    function startGame() {
```

Comment:
Does this need to be wrapped in a function?

Reply:
No, it doesn't. Good catch!